src/config/constants.js

```diff
@@ -244,3 +209,46 @@ export const DEFAULT_THERMOMETER_PERCENTAGE_FULL = 0.4;
 export const THERMOMETER_NUMBER_OF_GRADATIONS = 10;
 export const THERMOMETER_GRADATION_WIDTH = 0.25;
 export const THERMOMETER_GRADATION_STROKE = 1;
+
+export const SET_INTERVAL_TIME = 80;
+export const STROKE_COLOR = 'black';
+export const DEFAULT_TIMER_COUNT = 1;
+
+export const LINE_STROKE_COLOR = 'black';
+export const DEFAULT_TENSION = 0.3;
```

Comment:
might be helpful to clarify which tension this is referring to: `DEFAULT_RADIATION_LINE_TENSION`